websites/U/uwowocosplay/metadata.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Uwowocosplay",
+	"description": {
+		"en": "Uwowo is a professional cosplay costume brand focusing on the high quality anime and game costumes and wigs with affordable price."
+	},
+	"url": "uwowocosplay.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/OczeyDO.jpeg",
+	"thumbnail": "https://i.imgur.com/6091d7r.png",
+	"color": "#f17174",
+	"category": "other",
+	"tags": [
+		"other"
+	]
```

There are new fields called `matches` and `iFrameMatches`, which are required for new Presences. These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.

In your case only matches is required.
```suggestion
	], 
		"matches": [
		"*://uwowocosplay.com/*"
	]
```